Update Energy Card Documentation (#43730)
home-assistant/home-assistant.github.io: source/_dashboards/energy.markdown

This is a list of all the cards used in the energy dashboard. You can also place them anywhere you want in your dashboard.

> These cards can either be added using the visual card editor or manually using the YAML editor. In the visual editor, these are located in the **Energy cards** section of the card picker dialog.

Reviewer: ```suggestion
You can add these cards using the visual card editor or by editing the YAML directly. In the visual editor, these are located in the **Energy cards** section of the card picker dialog.
```

This card allows you to pick what data to show. Changing it in this card will influence the data in all other cards.
Specific dates and ranges can be selected by opening the date range picker. The current period can be compared to the previous one using the compare data option within the menu.

### YAML configuration

The following YAML options are available:

{% configuration %}
type:
  required: true
  description: "`energy-date-selection`"
  type: string
collection_key:
  required: false
  description: "Collection key to use for the card. This links the card to a specific energy dashboard collection. If not provided, defaults to the current dashboard page URL."
  type: string

> vertical_opening_direction:

Reviewer 2: Need to add `opening-direction` here as well

## Using multiple collections

By default, all energy cards on the current dashboard are linked together. Any `energy-date-selection` cards on this dashboard control what data is shown. If there are none, a default date of today is used. When you add multiple date selection cards, they always show the same date. Any `energy-date-selection` card on a different dashboard does not affect energy cards on the current dashboard.

> To enable multiple different date selections on the same dashboard, it is necessary to link them to different collections. This is done using the `collection_key` parameter either in the visual editor or card YAML, and giving this a value of any custom string that begins with `energy_` (strings that do not start with `energy_` will generate an error).

Reviewer: ```suggestion
To enable multiple different date selections on the same dashboard, they must be linked to different collections. This is done using the `collection_key` parameter, either in the visual editor or in the card YAML, with a value of any custom string that begins with `energy_` (strings that do not start with `energy_` will generate an error).
```
